### PR TITLE
Disambiguate account names when buying

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/routes/buy/info.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ class BuyInfo extends Component<Props, State> {
         )
         .then(results => results.filter(result => result))
         // @ts-ignore
-        .then(accounts => accounts.map(({ name, code }) => ({ text: name, value: code })))
+        .then(accounts => accounts.map(({ isToken, name, coinName, code }) => ({ text: isToken ? name : name + ' (' + coinName + ')', value: code })))
```
Comment (thisconnect) [inline frontends/web/src/routes/buy/info.tsx:90]: or `` `${name} (${coinName})` ``

**Review by thisconnect — APPROVED** [review]
LGTM

File: frontends/web/src/routes/buy/info.tsx (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/routes/buy/info.tsx:90]: On a fresh setup name and coinName will be the same should we still display coinName?

Screen Shot 2021-05-20 at 6 35 18 AM

Comment (thisconnect) [reply frontends/web/src/routes/buy/info.tsx:90]: Updated locize dropdown placeholder, from "Select a coin" to "Select an account"

Screen Shot 2021-05-20 at 6 40 45 AM

Comment (benma) [reply frontends/web/src/routes/buy/info.tsx]: I removed the entry altogether - why have almost the same text twice? Pls check the new commit.

Comment (benma) [reply frontends/web/src/routes/buy/info.tsx]: > On a fresh setup name and coinName will be the same should we still display coinName?

Great catch. Fixed.

File: frontends/web/src/routes/buy/info.tsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ class BuyInfo extends Component<Props, State> {
         )
         .then(results => results.filter(result => result))
         // @ts-ignore
-        .then(accounts => accounts.map(({ name, code }) => ({ text: name, value: code })))
+        .then(accounts => accounts.map(({ isToken, name, coinName, code }) => ({ text: isToken || name === coinName ? name : `${name} (${coinName})`, value: code })))
```
Comment (thisconnect) [inline frontends/web/src/routes/buy/info.tsx:90]: Some parentheses would help readability and the intention of the code see the difference:
- `` ({ text: isToken || name === coinName ? name : `${name} (${coinName})`, value: code }) ``
- `` ({ text: (isToken || name === coinName) ? name : `${name} (${coinName})`, value: code }) ``
- `` ({ text: isToken || (name === coinName ? name : `${name} (${coinName})`), value: code }) ``

File: frontends/web/src/routes/buy/info.tsx [hunk 3]
```diff
@@ -195,9 +195,8 @@ class BuyInfo extends Component<Props, State> {
                                     <h1 class={style.title}>{t('buy.title', { name })}</h1>
                                     <Select
                                         label={t('buy.info.selectLabel')}
-                                        placeholder={t('buy.info.selectPlaceholder')}
                                         options={[{
-                                                text: t('buy.info.selectPlaceholder'),
+                                                text: t('buy.info.selectLabel'),
```
Comment (thisconnect) [inline frontends/web/src/routes/buy/info.tsx:199]: @benma so the label will say: "Choose your account"
and the select placeholder also "Choose your account"
is that what you really really want?

File: frontends/web/src/routes/buy/info.tsx (hunk 2 above)
Comment (benma) [reply frontends/web/src/routes/buy/info.tsx:90]: i am an opponent of parens that are not required :O imho everyone should be familiar with precedence rules.

File: frontends/web/src/routes/buy/info.tsx (hunk 3 above)
Comment (benma) [reply frontends/web/src/routes/buy/info.tsx:198]: good point - removed the label as well :D

File: frontends/web/src/routes/buy/info.tsx (hunk 2 above)
Comment (thisconnect) [reply frontends/web/src/routes/buy/info.tsx:90]: I prefer readable code, when you write math you also write parentheses even tho it is not needed right?

File: frontends/web/src/routes/buy/info.tsx (hunk 3 above)
Comment (benma) [reply frontends/web/src/routes/buy/info.tsx]: i went back and now show `Select a coin` only. I feel it's more intuitive, you want to buy a coin, not an account.

Comment (thisconnect) [reply frontends/web/src/routes/buy/info.tsx]: sorry I changed it to "select an account" again :P 
let me know if I should redo.
lets discusss in chat ping @jadzeidan

**Pull request merged**